app/views/schools/add_participants/email_taken.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :title, "Email used" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% if email_used_in_the_same_school? %>
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-7">This email has already been added</h1>
+
+      <p class="govuk-body">This email address has already been used to add an early career teacher or mentor.</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>if they are an early career teacher, see if they have been added to another cohort</li>
+        <li>if they are a mentor, you just need <%= govuk_link_to "add them to this cohort", "#" %></li>
```

Comment:
Can we have a todo comment here (link to work in future ticket)

Reply:
I don't think we want to show this this year, right? Just confusing

Reply:
It is even more confusing as they could have been already added to this cohort :)